src/ucp/api/ucp.h

```diff
@@ -636,6 +636,10 @@ void ucp_worker_release_address(ucp_worker_h worker, ucp_address_t *address);
  * @li Transport layers, implementing asynchronous progress using threads,
  * require callbacks and other user code to be thread safe.
  *
+ * @note
+ * progress can be done implicitly by blocking functions, and will not be done
```

Comment:
The state of communication can be advanced (progressed) by blocking routines. Nevertheless, the non-blocking routines can not be used for communication progress